### PR TITLE
add zip lookup

### PR DESCRIPTION
**Pull request by pizzapanther**
more info about my code here

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ var apicache = require('apicache');
 var cache = apicache.middleware;
 
 var axios = require('axios');
+var pgp = require('pg-promise')();
```
Comment (pizzapanther) [inline app.js:7]: What does this do?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This is my app
+
+this is some text
```
Comment (pizzapanther) [inline README.md]: This sucks remove it!!!!!!

**Review by pizzapanther — COMMENTED** [review]
fix stuff please

File: README.md (hunk 2 above)
Comment (antweer) [reply README.md]: sry

File: app.js (hunk 1 above)
Comment (antweer) [reply app.js:7]: idk

Comment (pizzapanther) [reply app.js:7]: don't worry about it

**Pull request merged**